src/PresentationalComponents/TagsToolbar/TagsToolbar.js

```diff
@@ -1,6 +1,8 @@
 import './_TagsToolbar.scss';
 
+import { Divider, InputGroup, TextInput } from '@patternfly/react-core';
```

Comment:
checkout how we bring in the select component below, for the sake of small bundles, this needs to be dedicated imports for each piece

this also applies to line 5, the icons, lookin around the codebase yah should find examples of how this is done